### PR TITLE
Language and country selection modal

### PR DESCRIPTION
**Pull request by ankitecd**
Changes in this pull request:
- Added language and country selection modal opened from footer

### CODE REVIEW

File: bash.exe.stackdump [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Stack trace:
```
Comment (norbertschuler) [inline bash.exe.stackdump:1]: This surely should not be committed, should it?

File: src/utils/supportedLanguages.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "languageName": "Brazilian Portuguese"
+  },
+  {
+    "langCode": "de",
+    "languageName": "German"
+  },
+  {
+    "langCode": "sv",
+    "languageName": "Swedish"
+  }
```
Comment (norbertschuler) [inline src/utils/supportedLanguages.json:29]: If you don't want to translate the language name in every language later on, I recommend using the native language name here as I did to the Treecounter app here (without Swedish): https://github.com/Plant-for-the-Planet-org/treecounter-app/blob/175f7421d5aeb51a6adeabf0b263c720bbb361f3/app/components/Menu/index.js#L91-L98

File: src/utils/countriesData.json [hunk 3]
```diff
@@ -0,0 +1,1752 @@
+[
+  {
+    "countryName": "New Zealand",
+    "countryCode": "NZ",
+    "currencyName": "New Zealand Dollars",
+    "currencyCode": "NZD",
+    "currencyCountryFlag": "NZ"
+  },
```
Comment (norbertschuler) [inline src/utils/countriesData.json:8]: If you ever want to use translations for the country names, you need this file in every language we want to support. This is still an unsolved problem in the Treecounter app yet (but there are free sources on the internet providing this information).

File: bash.exe.stackdump (hunk 1 above)
Comment (ankitecd) [reply bash.exe.stackdump]: Yes it should not be. Deleted the file thanks :)

File: src/utils/countriesData.json (hunk 3 above)
Comment (harshvitra) [reply src/utils/countriesData.json:8]: We will add this later

**Pull request merged**